pandas/core/indexes/base.py

```diff
@@ -994,9 +994,13 @@ def to_series(self, **kwargs):
         """
 
```

Comment:
instead of this, simply list the kwargs that we accept, ``name=None, index=None`` and pass them thru.